quarkc/test/parse/literals.q

```diff
@@ -20,4 +20,5 @@ void literals() {
       2: "two",
       3: "three"};
     3.14149265359;
```

Comment:
This doesn't look like something you changed, but I can't help but notice that the floating point value is suspiciously close, but *not* actually pi. Pi is 3.141*5*9265359. ;-)